website/blog/2024/11-06-PromCon-EU-2024-md

```diff
@@ -0,0 +1,45 @@
+---
+title: PromCon EU 2024 Highlights
+linkTitle: PromCon EU 2024 Highlights
+newsSubtitle: November 06, 2024
+publishdate: 2024-11-01
+authors:
+- name: Christoph Kleineweber
+  email: c.kleineweber@sap.com
+  avatar: https://avatars.githubusercontent.com/u/318416
+- name: Jeremy Rickards
+  email: jeremy.rickards@sap.com
+  avatar: https://avatars.githubusercontent.com/u/9338170
+aliases: ["/blog/2024/11/06/01"]
+---
+
+
+Many innovative observability and application performance management (APM) products and services were released over the last few years. They often adopt or enhance concepts that Prometheus invented more than a decade ago. However, Prometheus, as an open-source project, has never lost its importance in this fast-moving industry and is the core of Gardener's monitoring stack.
+
+On September 11th and 12th, Prometheus developers and users met for PromCon EU 2024 in Berlin. The single-track conference provided a good mix of talks about the latest Prometheus development and governance by many core contributors, as well as users who shared their experiences with monitoring in a cloud-native environment. The overarching topic of the event was the upcoming Prometheus 3.0 release. Many of the presented innovations will also be interesting for the future of Gardener's observability stack. We will take a closer look at the changes in the Prometheus and Perses projects in the remainder of this article.
+
+![alt text](images/promcon2024.jpeg)
+
+## Prometheus 3.0 - OpenTelemetry Everywhere
+
+A first beta version for Prometheus 3.0 was released during the first day of the conference. The main focus of the new major release is compatibility with OpenTelemetry and full support for metric ingestion using the OpenTelemetry protocol (OTLP). While OpenTelemetry and Prometheus metrics have many things in common, a lot of incompatibilities still need to be sorted out.
+
+One of the differences can be found in the naming conventions for metric and label or attribute names. Instead of simply replacing dots with underscores, the Prometheus developers decided to introduce full UTF-8 support to achieve the best possible compatibility. This plan brought up interesting syntactical challenges for PromQL queries as well as a demand for conversion when interacting with systems that still have the restrictions of previous Prometheus releases.
```

Comment:
```suggestion
One of the differences can be found in the naming conventions for metrics and label or attribute names. Instead of simply replacing dots with underscores, the Prometheus developers decided to introduce full UTF-8 support to achieve the best possible compatibility. This plan brought up interesting syntactical challenges for PromQL queries, as well as a demand for conversion when interacting with systems that still have the restrictions of previous Prometheus releases.
```